pages/docs/tutorials/kafka/_section.md

```diff
@@ -1,4 +1,4 @@
 ---
 title: 'Kafka'
-weight: 20
+weight: 220
```

Comment:
@alequetzalli Any specific reason, why we moved the weight to such a high value `220`?

Reply:
yes, because that is how far we are now